src/test_l1_dsVideoDevice.c

```diff
@@ -1004,11 +1021,11 @@ void test_l1_dsVideoDevice_positive_dsForceDisableHDRSupport(void)
 
         // Step 03: Force disable HDR support (set to true)
         result = dsForceDisableHDRSupport(handle, true);
-        UT_ASSERT_EQUAL(result, dsERR_OPERATION_NOT_SUPPORTED);
+        UT_ASSERT_EQUAL(result, dsERR_NONE);
 
         // Step 04: Force disable HDR support (set to false)
         result = dsForceDisableHDRSupport(handle, false);
-        UT_ASSERT_EQUAL(result, dsERR_OPERATION_NOT_SUPPORTED);
+        UT_ASSERT_EQUAL(result, dsERR_NONE);
```

Comment:
as per spec changes is not valid 
https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsVideoDevice.h#L283

Reply:
updated